packages/opencensus-core/src/trace/instrumentation/base-plugin.ts

```diff
@@ -118,19 +121,28 @@ export abstract class BasePlugin implements types.Plugin {
    */
   private loadInternalFiles(): ModuleExportsMapping {
     let result: ModuleExportsMapping = null;
+    let overlap = false;
     if (this.internalFileList) {
       this.logger.debug('loadInternalFiles %o', this.internalFileList);
       Object.keys(this.internalFileList).forEach(versionRange => {
         if (semver.satisfies(this.version, versionRange)) {
+          if (result) {
```

Comment:
Not to be pedantic, but this can yield strange results if `result` just happens to be a falsey value (`0` or `''`). For correctness' sake I would just make `overlap` a counter (or otherwise just print the warning right in this function).

Reply:
Refactory done.